conference.js

```diff
@@ -2278,8 +2278,10 @@ export default {
      * @param {boolean} [requestFeedback=false] if user feedback should be
      * @param {string} [hangupReason] the reason for leaving the meeting
      * requested
+     * @param {boolean} [notifyOnConferenceTermination] whether to notify
+     * the user on conference termination
      */
-    hangup(requestFeedback = false, hangupReason) {
+    hangup(requestFeedback = false, hangupReason, notifyOnConferenceTermination) {
```

Comment:
This feels wrong since it will be easy for some part of the code to pass the wrong value. Why not check the config inside the function, when the reason si the right one?